INTERPRETING.md

```diff
@@ -0,0 +1,175 @@
+# Interpreting Test262 Tests
+
+All tests are declared as text files located within this project's `test`
+directory. In order to execute Test262 tests, runtimes must observe the
+following semantics.
+
+## Test Execution
+
+Test262 tests are only valid under the runtime environment conditions described
+here. Test environments may be further modified according to the metadata
+contained with each test--refer to the "Metadata" section for more details.
+
+### Realm Isolation
+
+Each test must be executed in a new [ECMAScript
+realm](https://tc39.github.io/ecma262/#sec-code-realms) dedicated to that test.
+Unless configured otherwise (via the `module` flag), source text must be
+interpreted as [global
+code](http://www.ecma-international.org/ecma-262/6.0/#sec-types-of-source-code).
+
+### Test262-Defined Bindings
+
+The contents of the following files must be evaluated in the test realm's
+global scope prior to test execution:
+
+1. `harness/assert.js`
+2. `harness/sta.js`
+
+### Host-Defined Functions
+
+The following values must be defined as writable, configurable, non-enumerable
+properties of the global scope prior to test execution.
+
+- **`print`** A function that exposes the string value of its first argument to
+  the test runner. This is used as a communication mechanism for asynchronous
+  tests (via the `async` flag, described below).
+
+### Strict Mode
+
+Unless configured otherwise (via the `noStrict`, `onlyStrict`, or `raw` flags),
+each test must be executed twice: once in ECMAScript's non-strict mode, and
+again in ECMAScript's strict mode.
```

Comment:
This doc needs to better explain what this means. According to the spec, strictness is a property of regions of source code, not an execution mode of the processor than can be enabled or disabled at will. See issue #299 and especially https://github.com/tc39/test262/issues/299#issuecomment-108620354 where I take a stab at defining what test262 means by "executing a test in (non-)strict mode".